packages/tools/client-debugger/client-debugger/src/messaging/TelemetryMessages.ts

```diff
@@ -30,4 +30,22 @@ export interface TelemetryEventMessage extends IDebuggerMessage<TelemetryEventMe
 	type: "TELEMETRY_EVENT";
 }
 
+/**
+ * Outbound event indicating that all telemetry events has been generated by the application.
+ * Includes all telemetry events.
+ *
+ * @public
+ */
+export interface AllTelemetryEventsMessage extends IDebuggerMessage<TelemetryEventMessageData> {
+	type: "ALL_TELEMETRY_EVENTS";
+}
```

Comment:
```suggestion
export interface TelemetryHistoryMessage extends IDebuggerMessage<TelemetryEventMessageData> {
	type: "TELEMETRY_HISTORY";
}
```
Nit, to make the relation between the "request" and the "response" messages clearer.